lib/tasks/config.js

```diff
@@ -34,7 +34,7 @@ class TaskConfig extends Task {
         'finishOnKey', 'input', 'numDigits', 'minDigits', 'maxDigits',
         'interDigitTimeout', 'bargein', 'dtmfBargein', 'minBargeinWordCount', 'actionHook'
       ].forEach((k) => {
-        if (this.bargeIn[k]) this.gatherOpts[k] = this.bargeIn[k];
+        if (this.bargeIn[k] !== undefined) this.gatherOpts[k] = this.bargeIn[k];
```

Comment:
I think
const val = this.bargeIn[k];
if (val !== undefined && val !== null) this.gatherOpts[k] = val;

what do you think?

Reply:
@xquanluu made changes please review